web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -244,14 +244,14 @@ const sortByName = (a: AssetStats, b: AssetStats) => {
 const EXTERNAL_SERVICES = [
   {
     name: 'Bioimagesuite/Viewer',
-    regex: '.nii(.gz)?$',
+    regex: '\.nii(\.gz)?$',
```

Comment:
```suggestion
    regex: /\.nii(\.gz)?$/,
```

Reply:
I have no clue in JS, so if you are saying this specification is somehow better - I would just trust you. But how or why is `/.../` form better than `'...'` -- does it compile it into regex or smth like that?

Reply:
`/.../` only works for literal values, whereas `new RegExp(...)` allows for dynamic values since it accepts regular strings. `/.../` is more performant, since it doesn't have the overhead of "converting" the string into a regex. There's more info here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#creating_a_regular_expression